systems-management/nisysmgmt.yml

```diff
@@ -0,0 +1,2030 @@
+      summary: Get systems summary
+      description: Get the number of connected and disconnected managed systems.
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/ManagedSystemsSummary'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/get-jobs-summary:
+    get:
+      operationId: GetJobsSummary
+      x-ni-privilege: Read
+      x-ni-operation: getJobs
+      tags: [jobs]
+      summary: Get jobs summary
+      description: Get the number of active, failed and succeeded jobs.
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/JobsSummary'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/configure-feeds:
+    post:
+      operationId: ConfigureAndUpdateMultipleFeeds
+      x-ni-privilege: Read
+      x-ni-operation: manageFeeds
+      tags: [feeds]
+      summary: Configure and update multiple feeds
+      description: Configure and update multiple feeds.
+      parameters:
+        - in: body
+          name: FeedUris
+          required: true
+          description: Feeds to be configured and updated.
+          schema:
+            type: array
+            items:
+              type: string
+              description: The URI of the feed.
+              example: 'http://download.ni.com/support/nipkg/products/ni-package-manager/released'
+      responses:
+        200:
+          $ref: '#/responses/PartialSuccess'
+        204:
+          description: No Content
+        400:
+          $ref: '#/responses/BadRequest'
+  /v1/get-discovered-systems-summary:
+    get:
+      operationId: GetDiscoveredSystemsSummary
+      x-ni-privilege: Read
+      x-ni-operation: getSystems
+      tags: [systems]
+      summary: Get discovered systems summary
+      description: Get the number of network discovered systems. This number refers to the count of systems that are discovered and not managed.
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/DiscoveredSystemsSummary'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/get-pending-systems-summary:
+    get:
+      operationId: GetPendingSystemsSummary
+      x-ni-privilege: Read
+      x-ni-operation: getSystems
+      tags: [systems]
+      summary: Get pending systems summary
+      description: Get the number of systems that are currently pending to be added to the master but have not yet been approved by the master.
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/PendingSystemsSummary'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/get-systems-report:
```

Comment:
The old API had a way of automatically uploading the systems software report to the file ingestion service and returning a file ID instead of the actual report contents. Do we plan to continue supporting this use case in the new API?

Reply:
Only the current AMQP API supports that so we will keep this functionality only in AMQP API.